test/rekt/features/broker/eventing_tls_feature.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2023 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package broker
+
+import (
+	"context"
+	"time"
+
+	cetest "github.com/cloudevents/sdk-go/v2/test"
+	"github.com/google/uuid"
+	"k8s.io/apimachinery/pkg/types"
+	"knative.dev/pkg/system"
+	"knative.dev/reconciler-test/pkg/eventshub"
+	"knative.dev/reconciler-test/pkg/eventshub/assert"
+	"knative.dev/reconciler-test/pkg/feature"
+	"knative.dev/reconciler-test/pkg/resources/service"
+	"knative.dev/reconciler-test/resources/certificate"
+
+	"knative.dev/eventing/test/rekt/features/featureflags"
+	"knative.dev/eventing/test/rekt/resources/addressable"
+	"knative.dev/eventing/test/rekt/resources/broker"
+	"knative.dev/eventing/test/rekt/resources/trigger"
+)
+
+func RotateIngressTLSCertificate() *feature.Feature {
+	ingressCertificateName := "mt-broker-ingress-server-tls"
+	ingressSecretName := "mt-broker-ingress-server-tls"
+
+	filterCertificateName := "mt-broker-filter-server-tls"
+
+	brokerName := feature.MakeRandomK8sName("broker")
+	triggerName := feature.MakeRandomK8sName("trigger")
+	sink := feature.MakeRandomK8sName("sink")
+	source := feature.MakeRandomK8sName("source")
+
+	f := feature.NewFeatureNamed("Rotate " + ingressCertificateName + " certificate")
+
+	f.Prerequisite("transport encryption is strict", featureflags.TransportEncryptionStrict())
+	f.Prerequisite("should not run when Istio is enabled", featureflags.IstioDisabled())
+
+	f.Setup("Rotate ingress certificate", certificate.Rotate(certificate.RotateCertificate{
+		Certificate: types.NamespacedName{
+			Namespace: system.Namespace(),
+			Name:      ingressCertificateName,
+		},
+	}))
+	// We cannot externally verify this certificate rotation
+	f.Setup("Rotate filter certificate", certificate.Rotate(certificate.RotateCertificate{
```

Comment:
Why is the filter certificate rotated as well?

Reply:
Because why not?

Reply:
> Because why not?

I don't understand, how this relates to the ingress cert rotation :thinking:

Reply:
Ok, let me rename the test... to `TestBrokerRotateTLSCertificate`

Reply:
Renamed to `TestMTChannelBrokerRotateTLSCertificates` https://github.com/knative/eventing/pull/7103/commits/478d813951e2243c4821c488379f8023e62aa3e3

Reply:
Ah ok :+1: 
/lgtm